nixos/modules/config/system-path.nix

```diff
@@ -11,7 +11,7 @@ let
     [ config.nix.package
       pkgs.acl
       pkgs.attr
-      pkgs.bashInteractive # bash with ncurses support
+      pkgs.bash.interactive # bash with ncurses support
```

Comment:
as the old names (```bashInteractive```) are going to be broken, maybe it is time to invent a more precise name which would not need to be explained?

```bash.withNcurses``` ?